libvast/src/concept/parseable/vast/expression.cpp

```diff
@@ -56,13 +56,69 @@ static predicate::operand to_data_operand(data x) {
   return x;
 }
 
-static predicate to_value_predicate(data x) {
+// Expands a predicate with a type extractor, an equality operator, and a
+// corresponding data instance according to the rules of the expression
+// language.
+struct expander {
+  expression operator()(caf::none_t) const {
+    return expression{};
+  }
+  expression operator()(const conjunction& c) const {
```

Comment:
Why aren't there empty lines between the member functions?

Reply:
Fixed.